EZSwiftExtensionsTests/UIFontTests.swift

```diff
@@ -13,7 +13,7 @@ class UIFontTests: XCTestCase {
     
     let TEST_SIZE = CGFloat(10)
     
-    let TEST_FONT_TYPE = FontType.Bold
+    let TEST_FONT_TYPE = FontType.bold
```

Comment:
Sure.

https://swift.org/documentation/api-design-guidelines/#follow-case-conventions
This guideline says `Names of types and protocols are UpperCamelCase. Everything else is lowerCamelCase.`

swift-evolution: Decision Notes
https://lists.swift.org/pipermail/swift-evolution-announce/2016-March/000053.html